experimental/gittuf/common.go

```diff
@@ -0,0 +1,17 @@
+// Copyright The gittuf Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package gittuf
+
+import "github.com/gittuf/gittuf/internal/gitinterface"
+
+// CheckIfSigningViable checks if we are able to sign RSL entries given the
+// current environment
+func CheckIfSigningViable() error {
```

Comment:
```suggestion
func CheckIfGitSigningEnabled() error {
```

Reply:
Should this be a method on Repository in this package?

Reply:
I meant should it be a method on `gittuf.Repository`, not `gitinterface.Repository`.

Reply:
Using `r.r.CanSign` makes a lot of sense fwiw.